backend/src/consoles/SSHConsole.ts

```diff
@@ -91,8 +96,13 @@ export default class SSHConsole extends EventEmitter implements Console {
       stream
         .on("close", () => {
           console.log("Stream :: close");
+          stream.end();
+          sshConsole.end();
           this.emit("close");
         })
+        .on("error", (err: Error) => {
+          console.log("Error handler!", err);
```

Comment:
Error handler! is not a good logging message, just logging the error would be sufficient due to the stack trace. This appears more often, I guess I introduced that at some point.

Reply:
;) Yea... I just copied that from your code, but your right, not a good label. Changed it and also clarified other output to relate back to where it belongs to. Maybe this would have sped up my session with debugger and code analysis to fix the ssh connection leaks ;)